api/token/route.js

```diff
@@ -0,0 +1,2 @@
+// DO NOT REMOVE. This file is used in the build and deploy Storybook on Vercel.
+// @TODO: look for more elegant solution
```

Comment:
Establishing the serverless function on Vercel for Storybook setup is a bit tricky.
First of all, there is an official instruction: [Deploying Storybook with Vercel](https://vercel.com/guides/storybook-with-vercel).
Secondly, [Serverless Functions Quickstart](https://vercel.com/docs/functions/serverless-functions/quickstart).
The thing is that seems like Vercel runs some simple HTTP server for the Storybook setup, which can’t handle unbundled files (e.g. *.ts).
So, I’ve seen two main roads here:
1. Configure a dummy Next.js project to handle calls to the functions and serve static Storybook content.
2. Find a way to force Vercel’s Storybook env to handle function calls.
I didn’t find a quick solution for 1 and honestly, I don’t like the idea of proxying calls to static files through the next.js env.
I decided to follow 2. The initial assumption was to pre-build a script for the serverless function with tsc to the vercel’s `api/` folder (Vercel’s reserved folder). The build workflow works without errors locally and on Vercel, but for some reasons, Vercel refuses to include tsc output in the final Deployment.
What helped was the creation of an empty file (a placeholder) and adding it to the repo, e.g. `api/token/route.js`.
With this hack Vercel started to include the function to the Deployment.